api/app/tests/apps/email_integrations/test_mail.py

```diff
@@ -20,8 +21,34 @@
 from signals.apps.signals.models import Note
 
 
-class TestMailActionTriggers(TestCase):
+class BaseTestMailCase(TestCase):
     def setUp(self):
+        EmailTemplate.objects.create(key=EmailTemplate.SIGNAL_CREATED,
+                                     title='Uw melding {{ signal_id }}',
+                                     body='{{ text }} {{ created_at }} {{ handling_message }} {{ ORGANIZATION_NAME }}')
```

Comment:
I think it is intentional, but just to confirm: 

The `handling_message` is the new intended name, and `afhandelings_text` that is currently (and in this PR still) present in the reporter_rules is there for backwards compatibility with existing mail templates (as stored in DB)?

Reply:
Correct, `afhandelings_text` is still present in the context so that it is backwards compatible. I just kept all the items we put on the context so that we can have a transition period. It will be removed later (SIG-3634)